tests/snowpark/test_jobs.py

```diff
@@ -10,32 +10,37 @@ def test_create_job(mock_connector, runner, mock_ctx):
     ctx = mock_ctx()
     mock_connector.return_value = ctx
 
-    with NamedTemporaryFile(
-        prefix="spec", suffix="yaml", dir=Path(__file__).parent
-    ) as fh:
-        name = fh.name
-        result = runner.invoke(
+    test_spec = """
+spec:
+  containers:
+  - name: main
+    image: public.ecr.aws/myrepo:latest
+    """
+    with TemporaryDirectory() as temp_dir:
+        filepath = os.path.join(temp_dir, "test")
+        with open(filepath, "w") as fh:
+            fh.write(test_spec)
+        runner.invoke(
             [
                 "snowpark",
                 "jobs",
                 "create",
                 "--compute-pool",
-                "jobName",
+                "testPool",
                 "--spec-path",
-                fh.name,
-                "--stage",
-                "stageValue",
+                filepath,
             ]
         )
-
-    assert result.exit_code == 0, result.output
+        with open("/Users/aivanou/code/snowcli-aivanou/test.txt", "w") as fh:
```

Comment:
This won't work on CI. Also what's exactly the purpose of it?

Reply:
oh, apologies